decidim-elections/app/controllers/decidim/votings/admin/votings_controller.rb

```diff
@@ -80,6 +80,30 @@ def unpublish
           end
         end
 
+        def available_polling_officers
+          respond_to do |format|
+            format.json do
+              if (term = params[:term].to_s).present?
+                query = current_voting
```

Comment:
maybe we can extract this to a query object, what do you think?

Reply:
ok, sorry, I see that you already moved to the model 👍

Reply:
Yes! Was planning on doing this but then got carried away with other things. Thanks!